app/subscribers/proxy_config_event_subscriber.rb

```diff
@@ -3,7 +3,9 @@
 class ProxyConfigEventSubscriber
   def call(event)
     case event
-    when ProxyConfigs::AffectingObjectChangedEvent then ProxyConfigAffectingChangeWorker.perform_later(event.event_id)
+    when ProxyConfigs::AffectingObjectChangedEvent then
```

Comment:
Rubocop complains about this `then` here

```suggestion
    when ProxyConfigs::AffectingObjectChangedEvent
```